flink-ml-core/src/test/java/org/apache/flink/ml/api/StageTest.java

```diff
@@ -463,5 +463,10 @@ public void testValidators() {
         Assert.assertTrue(nonEmptyArray.validate(new String[] {"1"}));
         Assert.assertFalse(nonEmptyArray.validate(null));
         Assert.assertFalse(nonEmptyArray.validate(new String[0]));
+
+        ParamValidator<String[]> isSubArray = ParamValidators.isSubArray("a", "b", "c");
+        Assert.assertFalse(isSubArray.validate(new String[] {"c", "v"}));
```

Comment:
nits: could we also check the case where the input value is null, to be consistent with the above tests?

`Assert.assertFalse(isSubArray.validate(null))`